packages/navigator/src/browser/navigator-frontend-module.ts

```diff
@@ -35,8 +35,9 @@ import { NavigatorDiff } from './navigator-diff';
 import { NavigatorLayoutVersion3Migration } from './navigator-layout-migrations';
 import { NavigatorTabBarDecorator } from './navigator-tab-bar-decorator';
 import { TabBarDecorator } from '@theia/core/lib/browser/shell/tab-bar-decorator';
+import { NavigatorWidgetFactory } from './navigator-widget-factory';
 
-export default new ContainerModule(bind => {
+export default new ContainerModule((bind, unbind, isBound, rebind) => {
```

Comment:
why do you need unbind, isBound and rebind?

Reply:
Leftovers from my testing. removed